### PR TITLE
LPS-78865 Adds to test

### PR DESCRIPTION
**Pull request by brunobasto**
@marcellustavares

### CODE REVIEW

**Comment by brunobasto** [comment]
ci:test

**Comment by brunobasto** [comment]
ci:test

**Comment by marcellustavares** [comment]
ci:test

**Comment by marcellustavares** [comment]
@brunobasto, let me fix the CI issue before merging this pull ok?

Thanks

**Comment by marcellustavares** [comment]
ci:test

**Comment by marcellustavares** [comment]
CI fixed at https://github.com/liferay/com-liferay-analytics/pull/63

File: analytics-client-js/test/manual/test.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h1>Test</h1>
 		<a href="https://bbc.co.uk">External link</a>
 	</p>
 
-	<form data-analytics data-analytics-form-id="my-analytics-form" id="form1" name="abc" action="http://google.com">
+	<form data-analytics data-analytics-form-id="my-analytics-form" data-analytics-title="My Nice Form" id="form1" name="abc" action="http://google.com">
```
Comment (marcellustavares) [inline analytics-client-js/test/manual/test.html]: @brunobasto, can we rename these properties to `data-analytics-asset-id` and `data-analytics-asset-title` so we can reuse to other assets?

**Pull request merged**